test/e2e/constants/constants.go

```diff
@@ -36,8 +36,8 @@ var (
 	RunSaOperator        = strings.ToUpper(constants.GetEnvWithDefault("RUN_SA_OPERATOR", "false"))
 	OperatorUpgradeStage = strings.ToUpper(constants.GetEnvWithDefault("OPERATOR_UPGRADE_STAGE", OperatorUpgradeStageNone))
 	CleanupInfinispan    = strings.ToUpper(constants.GetEnvWithDefault("CLEANUP_INFINISPAN_ON_FINISH", "true"))
-	ExpectedImage        = constants.GetEnvWithDefault("EXPECTED_IMAGE", "infinispan/server:12.0")
-	NativeImageName      = constants.GetEnvWithDefault("NATIVE_IMAGE", "infinispan/server-native:12.0")
+	ExpectedImage        = constants.GetEnvWithDefault("EXPECTED_IMAGE", "quay.io/infinispan/server:12.0")
```

Comment:
@rigazilla do we need to use images from quay.io in all places, i.e. deployment, test, etc?

Reply:
@dmvolod images are on docker and quay, not a requirement but quay should be preferable.
Actually this is just a test if CI is happier with quay, I've remembered that time ago I've got  a popup performance warning from Travis UI about pulling images from docker

Reply:
Oh, @rigazilla yes, but we should update default Infinispan image for operator also to prevent this problem
Will try to test with quay based images

Reply:
yeah, let's see if CI is happy with quay. If yes we can align all the code to pull from there

Reply:
hey this PR turned to green...

Reply:
Yeah, but tests still using images from docker hub. This means that CI is not stable anymore
```
{},"ready":false,"restartCount":0,"image":"docker.io/infinispan/server:12.0","imageID":"docker.io/infinispan/server@sha256:8696e77b3b866a938b205a65d5b4122b2d22d34769efc2fb87addf5fa0a52819","containerID":"containerd://7a5e416285a3b20e31c5740b0fc914dad7489fd0c05021e0caef0577b05113be","started":true}]}
```